SurveyApp/Views/AnswerView/DisplayType.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  DisplayType.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 7/6/21.
+//
+
+import UIKit
+
+enum DisplayType {
```

Comment:
This is a global variable, let's give it a more explanatory name `AnswerDisplayType` or something else?